docs/api.md

```diff
@@ -0,0 +1,366 @@
+[Documentation Home](./README.md)
+
+# API
+
+- [Generate a Prompt](#generate-a-prompt)
+- [Create a Model](#create-a-model)
+- [List Local Models](#list-local-models)
+- [Copy a Model](#copy-a-model)
+- [Delete a Model](#delete-a-model)
+- [Pull a Model](#pull-a-model)
+
+## Generate a Prompt
+
+**POST /api/generate**
+
+### Description
+
+**Generate** is the main endpoint that you will use when working with Ollama. This is used to generate a response to a prompt sent to a model.
+
+### Request
+
+The **Generate** endpoint takes a JSON object with the following fields:
+
+```JSON
+{
+  "model": "modelname",
+  "prompt": "prompt",
+}
+```
+
+### Response
+
+The response is a stream of JSON objects with the following fields:
+
+```JSON
+{
+  "model": "modelname",
+  "created_at": "2023-08-04T08:52:19.385406455-07:00"
+  "response": "the current token",
+  "done": false
+}
+```
+
+The final response in the stream also includes the context and what is usually seen in the output from verbose mode. For example:
+
+```JSON
+{
+  "model":"orca",
+  "created_at":"2023-08-04T19:22:45.499127Z",
+  "done":true,
+  "total_duration":5589157167,
+  "load_duration":3013701500,
+  "sample_count":114,
+  "sample_duration":81442000,
+  "prompt_eval_count":46,
+  "prompt_eval_duration":1160282000,
+  "eval_count":113,
+  "eval_duration":1325948000
+}
+```
+
+| field                | description                                |
+| -------------------- | ------------------------------------------ |
+| model                | the name of the model                      |
+| created_at           | the time the response was generated        |
+| response             | the current token                          |
+| done                 | whether the response is complete           |
+| total_duration       | total time spent generating the response   |
```

Comment:
This and other duration fields should they're measured in nanoseconds

Reply:
ok